yarn-project/acir-simulator/package.json

```diff
@@ -36,7 +36,8 @@
     "@noir-lang/acvm_js": "0.26.1",
     "levelup": "^5.1.1",
     "memdown": "^6.1.1",
-    "tslib": "^2.4.0"
+    "tslib": "^2.4.0",
+    "viem": "^1.2.5"
```

Comment:
I think you could have this in the devDependencies only? Are you not only using it in the test?